prebid-mobile/modules/rendering/android-sdk-customization-controls.md

```diff
@@ -0,0 +1,68 @@
+---
+
+layout: page_v2
+title: Prebid Mobile Rendering Modules
+description: Prebid Mobile Rendering Modules architecture
+sidebarType: 2
+
+---
+
+# Customization Controls
+
+Prebid SDK provides an API way to customize its behaviour. 
+
+> NOTE: In the nearest future the Server Side Configuration will be supported as well. Follow this [feature request](https://github.com/prebid/prebid-server/issues/2186) for the details. 
+
+
+## Rendering Controls
+
+The following properties allow to customize the rendering of Video Interstitial Ads.
+
+### Max Video Duration
+
+The `maxVideoDuration` indicates the maximum available playback time in seconds.
+If the value in the **Duration** tag is bigger than the given value SDK will fail to load ad, providing a respective error message.
+
+### Application Muted
+
+The `isMuted` property indicates whether the ad should run playback with sound or not.
+Default value - **false**.
+
+### Close Button Area
+
+The `closeButtonArea` property indicates the area which the close button should occupy on the screen. The possible values are from **0** to **1**.
```

Comment:
"area" is somewhat confusing term, can we clarify more by adding something like "percent of device screen" instead of area.